api/v1alpha1/manifest_types.go

```diff
@@ -172,14 +89,26 @@ const (
 // Manifest is the Schema for the manifests API.
 type Manifest struct {
 	metav1.TypeMeta   `json:",inline"`
-	metav1.ObjectMeta `json:"metadata"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
 
 	// Spec specifies the content and configuration for Manifest
-	Spec ManifestSpec `json:"spec"`
+	Spec ManifestSpec `json:"spec,omitempty"`
```

Comment:
is there a use case of a Manifest CR with an no spec?

Reply:
I mainly updated these fields to conform with Kubernetes defaults, see Deployment: https://github.com/kubernetes/api/blob/master/apps/v1/types.go#L362